arkouda_cws_hashing/test/cws_unit_test.chpl

```diff
@@ -0,0 +1,85 @@
+use CTypes;
+require '-lgsl','-lgslcblas';
+
+
+extern {
+#include "gsl/gsl_rng.h"
+#include "gsl/gsl_randist.h"
+}
+
+
+proc cantorPairing(setEltIdx: int(64), hashIdx: int(64)): real(64) {
+
+    var sum: int(64) = setEltIdx + hashIdx;
+
+    var result: real(64) = (0.5 * sum * (sum + 1)) + hashIdx;
+
+    return result;
+}
+
+
+var offsets: [0..10] int(64) = [0, 3, 10, 15, 27, 31, 38, 40, 44, 49, 50];
+var setElts: [0..49] int(64) = [3, 4, 1, 3, 7, 8, 2, 1, 8, 4, 2, 5, 6, 9, 2, 5, 3, 2, 8, 8, 9, 1, 5, 3, 9, 3, 3, 3, 1, 8, 1, 6, 9, 9, 8, 7, 3, 5, 1, 1, 3, 8, 8, 6, 4, 6, 5, 1, 7, 1];
+var weights: [0..49] real(64) = [0.333333, 0.25, 1.0, 0.333333, 0.142857, 0.125, 0.5, 1.0, 0.125, 0.25, 0.5, 0.2, 0.166667, 0.111111, 0.5, 0.2, 0.333333, 0.5, 0.125, 0.125, 0.111111, 1.0, 0.2, 0.333333, 0.111111, 0.333333, 0.333333, 0.333333, 1.0, 0.125, 1.0, 0.166667, 0.111111, 0.111111, 0.125, 0.142857, 0.333333, 0.2, 1.0, 1.0, 0.333333, 0.125, 0.125, 0.166667, 0.25, 0.166667, 0.2, 1.0, 0.142857, 1.0];
+
+
+var numHashes: int(64) = 3; 
+
+var preimages: [0..29] int(64);
+var minHashes: [0..29] real(64);
+
+
+for offsetIdx in {offsets.domain.first..offsets.domain.last-1} {
+
+    var setStart: int(64) = offsets[offsetIdx];
+    var setFinal: int(64) = offsets[offsetIdx+1];
+
+    for hashIdx in 0..numHashes-1 {
+
+        var outIdx: int(64) = offsetIdx*numHashes + hashIdx;
```

Comment:
should this test call `getMinHashes` instead of duplicating the logic?

Reply:
Oh, ignore that!  That doesn't need to be pulled.  It was just a helper program that I wrote to generate the output for my unit test.